xarray/tests/test_backends_datatree.py

```diff
@@ -62,6 +63,52 @@ def test_netcdf_encoding(self, tmpdir, simple_datatree):
         with pytest.raises(ValueError, match="unexpected encoding group.*"):
             original_dt.to_netcdf(filepath, encoding=enc, engine=self.engine)
 
+    def test_open_datatree(self):
```

Comment:
I could always add more tests for `open_datatree` but I think @shoyer really covered everything in `test_datatree.py`